testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/sso/remote/InfinispanServerSetupTask.java

```diff
@@ -36,9 +38,7 @@
     public InfinispanServerSetupTask() {
         this.builder.node(AbstractClusteringTestCase.TWO_NODES)
                 .setup("/socket-binding-group=standard-sockets/remote-destination-outbound-socket-binding=infinispan-server:add(port=%d,host=%s)", INFINISPAN_SERVER_PORT, INFINISPAN_SERVER_ADDRESS)
-// Disable infinispan-server security until test driver issues are fixed
-//                .setup("/subsystem=infinispan/remote-cache-container=sso:add(default-remote-cluster=infinispan-server-cluster, module=org.wildfly.clustering.web.hotrod, properties={infinispan.client.hotrod.auth_username=testsuite-user,infinispan.client.hotrod.auth_password=testsuite-password}, protocol-version=%s)", INFINISPAN_SERVER_PROTOCOL_VERSION)
-                .setup("/subsystem=infinispan/remote-cache-container=sso:add(default-remote-cluster=infinispan-server-cluster, module=org.wildfly.clustering.web.hotrod, protocol-version=%s)", INFINISPAN_SERVER_PROTOCOL_VERSION)
+                .setup("/subsystem=infinispan/remote-cache-container=sso:add(default-remote-cluster=infinispan-server-cluster, module=org.wildfly.clustering.web.hotrod, properties={infinispan.client.hotrod.auth_username=%s, infinispan.client.hotrod.auth_password=%s}, protocol-version=%s)", INFINISPAN_APPLICATION_USER, INFINISPAN_APPLICATION_PASSWORD, INFINISPAN_SERVER_PROTOCOL_VERSION)
```

Comment:
Do we need to specify the protocol version any longer?

Reply:
The original idea was to eventually test against different datagrid versions requiring different protocol versions. But with the protocol negotiation after 3.0 in place, we should not need this any longer. I ll remove.